### PR TITLE
Feature: Development script for package.json updates

### PR DESCRIPTION
**Pull request by srzainab**
### Summary of changes
- Updated outdated packages in package.json
- Added a `generatePackageJson.js` script
- Added husky hooks for generating and reverting dev package.json 
- Added reversion task as preDebugTask in launch.json
- Added `PULLFLOW_DEV_APP_URL` for avoiding conflicts with production app 
- Fixed formatting 

### Config changes
- Env variables
``` PULLFLOW_DEV_APP_URL=(string)```

### CODE REVIEW

**Review by hassan-pullflow — APPROVED** [review]
Looks good to me 👏 We will need to update development guidelines in `READ.me` for env change.

**Comment by srzainab** [comment]
I’m updating the readme @hassan-pullflow

File: src/commands/reconnect.ts [hunk 1]
```diff
@@ -29,7 +29,7 @@ export const Reconnect = async (
   if (codeReviews.requireRelogin) {
     log.error(codeReviews.error, module)
     window.showInformationMessage(`Pullflow: Please login again`)
-    commands.executeCommand(Command.signOut)
+    commands.executeCommand(Command('signOut'))
```
Comment (zakmandhro) [inline src/commands/reconnect.ts:32]: aside: the window message should be "Pullflow: Please sign in again."

**Review by zakmandhro — APPROVED** [review]
I don't fully understand this though

**Comment by zakmandhro** [comment]
Is the `generatePackageJson.js` file a convention for VS Code extensions?

**Pull request closed without merging**